versioned_docs/version-8.7/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -132,17 +132,92 @@ steps:
+| -------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `IDENTITY_TENANTS_0_NAME`                    | A human-readable name for the tenant.                                                                                                                                                                                                              | Default       |
+| `IDENTITY_TENANTS_0_TENANTID`                | The tenant ID. The `<default>` tenant is automatically created during Identity startup.                                                                                                                                                            | `<default>`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_TYPE`          | The type of member to add to the tenant, one of `GROUP`, `USER`, or `APPLICATION`. Additional members can be added by incrementing the second `0` value (for example, `IDENTITY_TENANTS_0_MEMBERS_1_TYPE`.)                                        | `APPLICATION` |
+| `IDENTITY_TENANTS_0_MEMBERS_0_APPLICATIONID` | If adding an **application** as a member of the tenant, the application ID. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_APPLICATIONID`). | `zeebe`       |
+| `IDENTITY_TENANTS_0_MEMBERS_0_USERNAME`      | If adding a **user** as a member of the tenant, the username. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_USERNAME`).                    | `user-name`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_GROUPNAME`     | If adding a **group** as a member of the tenant, the name of the group. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_GROUPNAME`).         | `group-name`  |
+| `IDENTITY_ENVIRONMENT_TENANTS_0_NAME`        | Needs description.                                                                                                                                                                                                                                 | Default       |
+| `IDENTITY_ENVIRONMENT_TENANTS_0_TENANTID`    | Needs description.                                                                                                                                                                                                                                 | `<default>`   |
```

Comment:
I could not find any description for the `ENVIRONMENT` variables.

Reply:
The tenants listed under `ENVIRONMENT` will be created (or if they exist, updated) by Identity on startup, according to the specified name and ID.

Reply:
I see, that makes sense. Thank you for the explanation.